schematic/manifest/generator.py

```diff
@@ -1367,16 +1370,20 @@ def set_dataframe_by_url(
         Returns:
             ps.Spreadsheet: A Google Sheet object.
         """
-
         # authorize pygsheets to read from the given URL
         gc = ps.authorize(custom_credentials=self.creds)
 
         # open google sheets and extract first sheet
         # This sheet already contains headers.
         sh = gc.open_by_url(manifest_url)
         wb = sh[0]
-
-        wb.set_dataframe(manifest_df, (1, 1), fit=True)
+        try:
```

Comment:
I think this is a perfect spot for a comment. I think Schematic tends to be over-commented, but this is somewhere I think warrants it. A comment explaining why this is needed would be great here.

Reply:
thanks, I think I added a comment.